sdk/examples/xo_python/sawtooth_xo/xo_client.py

```diff
@@ -30,8 +30,21 @@
 from sawtooth_xo.xo_exceptions import XoException
 
 
-def _sha512(data):
-    return hashlib.sha512(data).hexdigest()
+# namespace
+def _hash_name(name):
+    return hashlib.sha512(name.encode('utf-8')).hexdigest()
+
+FAMILY_NAME = 'xo'
```

Comment:
Hard-coding FAMILY_NAME and XO_NAMESPACE would make it more difficult to re-use this code to run xo games against another namespace.  In previous iterations, games could be played using an arbitrary namespace.